gtk/gtk-play.c

```diff
@@ -1253,6 +1299,8 @@ main (gint argc, gchar ** argv)
   gst_player_play (play.player);
   play.current_uri = g_list_first (play.uris);
 
+  g_timeout_add_seconds (5, (GSourceFunc) gtk_hide_toolbar_loop, &play);
```

Comment:
See g_source_remove() and the return value of g_timeout_add_seconds() for removing btw